README.md

```diff
@@ -1 +1,18 @@
-# extract-id3-tags-from-mp3s
+# Extract ID3 Tags From MP3s
+This program extracts id3 tags from an mp3 file, allowing automated metadata collection
+
+## Install
+Instructions on how to install
+
+## Usage
+Instructions on how to use
+
+## Contribute
+If you'd like to contribute, edit, or copy them, feel free to do so!
+
+Questions? We have a [forum](https://github.com/RichardLitt/extract-id3-tags-from-mp3s/issues/2) for open discussion! You can also open a new request if you'd like.
```

Comment:
Just say 'Open an issue'. I don't know why you're linking to a single issue in the forum - and it's not properly a forum, it's an issue tracker.